stdlib/3/urllib/request.pyi

```diff
@@ -1,12 +1,11 @@
-# Stubs for urllib.request (Python 3.4)
-
 import os
 import ssl
 import sys
 from email.message import Message
 from http.client import HTTPMessage, HTTPResponse, _HTTPConnectionProtocol
 from http.cookiejar import CookieJar
 from typing import IO, Any, Callable, ClassVar, Dict, List, Mapping, NoReturn, Optional, Sequence, Tuple, TypeVar, Union, overload
+from urllib.error import HTTPError as HTTPError
```

Comment:
I wasn't sure I agreed with this, but I see previously mentioned my objection: https://github.com/python/typeshed/issues/2149#issuecomment-421140124

Reply:
I'll revert this and instead filed https://github.com/python/cpython/pull/22304.